application/backend/src/main/resources/apps/typerefinery/components/graphs/barchart/template.html

```diff
@@ -2,6 +2,8 @@
     <div 
         class="barChart"
         id="barChartContainer"
+        data-path="${resource.path}"
+        data-model="${model.jsonString}"
         data-source="${model.dataSource}"
         data-model="${model.jsonString}"
```

Comment:
why data-model attribute added for two times.

Reply:
Done